pkg/kubelet/util/util_windows_test.go

```diff
@@ -317,3 +317,48 @@ func TestNormalizePath(t *testing.T) {
 		assert.Equal(t, test.normalizedPath, NormalizePath(test.originalpath))
 	}
 }
+
+func TestLocalEndpoint(t *testing.T) {
+	tests := []struct {
+		path             string
+		file             string
+		expectError      bool
+		expectedFullPath string
+	}{
+		{
+			path:             "/var/lib/kubelet/pod-resources",
+			file:             "kubelet",
+			expectError:      false,
+			expectedFullPath: `npipe://\\.\pipe\kubelet\pod-resources`,
```

Comment:
I don't think this is techinically wrong but we typically just specify it like the following: https://github.com/containerd/containerd/blob/71a7234f67feec950bbc25b9d89983d80d72a743/defaults/defaults_windows.go#L38

Reply:
I see no problem changing the format. Would something like `\\.\pipe\kubelet-pod-resources` acceptable?

Reply:
yea, I think that would make it consistent with alot of other tools

Reply:
changed, thanks!

Reply:
this looks the same?

Reply:
@ffromani oh, I see, I meant `npipe://` isn't really needed.  and would remove some of the duplicate slashes 🤷

Reply:
thanks for pointing this out. I can remove the `npipe://`. Which slashes are duplicate? according to the spec it seems to me that for example the leading `\\` is expected

Reply:
@jsturtevant there's something I'm not sure about. On unix, the function returns the path prefixed by `unix://` - so it looks like an URL more than a naked path. And indeed the code expects the protocol part (see https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/util/util_windows.go#L92) so it seems to me we should do the same here? Perhaps adding forward AND back slash leads to some confusion, but I'm not sure what we should use here. I do see the kubelet using paths with only slashes:
```
I0310 14:25:08.155586    3988 flags.go:64] FLAG: --container-runtime-endpoint="npipe:////./pipe/containerd-containerd"
```
(from https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/115133/pull-kubernetes-e2e-capz-windows-containerd/1634188079650574336/artifacts/clusters/capz-conf-jq6aid/machines/capz-conf-jq6aid-md-win-6d7c774d58-klznw/kubelet.log)